.changeset/swift-flies-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components': minor
```

Comment:
should this be a patch - a minor should offer new (non-breaking) functionality but does this do that?

```suggestion
'@guardian/source-react-components': patch
```